daemon/cmd/daemon.go

```diff
@@ -137,6 +138,8 @@ type Daemon struct {
 	identityAllocator *cache.CachingIdentityAllocator
 
 	k8sWatcher *watchers.K8sWatcher
+
+	hubbleServer *hubbleServer.LocalObserverServer
```

Comment:
Would you mind giving this struct member a more explicit name such as `hubbleObserver` or so? As it, it might be confused with hubble server (the gRPC server currently in hubble's `cmd/serve`.

Reply:
+1